apps/ui/src/helpers/efp.ts

```diff
@@ -0,0 +1,17 @@
+import { isAddress } from '@ethersproject/address';
+
+const EFP_API_URL = 'https://api.ethfollow.xyz';
+
+export async function getUserStats(
+  addressOrENS: string
+): Promise<{ followers_count: number; following_count: number }> {
+  if (!isAddress(addressOrENS)) return Promise.reject('Invalid address');
```

Comment:
Bit confusing that we named param as `addressOrENS` and rejecting if it is not address 
maybe we can rename param to `address`